src/client_handler.rs

```diff
@@ -808,7 +808,7 @@ impl ClientHandler {
                 destination,
                 amount,
                 transaction_id,
-            } => self.handle_transfer_money_vault_req(
+            } => self.handle_network_at2_bank_update(
```

Comment:
I just noticed, this got the same name for the two methods (client/vault). Do you have some preference in the distinction of them?